meshery

```diff
@@ -61,7 +61,32 @@ printf "\nExtracting %s...\n" "$NAME"
 unzip ${PWD}/meshery.zip 
 
 printf "\nInstalling mesheryctl in /usr/local/bin.\n"
-mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl
+if mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl ; then
+  echo "mesheryctl installed"
+else
+	echo "Cannot install mesheryctl. Aborting."
+  exit 1
+fi
+
+#Transfering permissions and ownership to client USER
+if [ "$SUDO_USER" == "" ] ;
+then
+	if 	chown $WHOAMI:$WHOAMI /usr/local/bin/mesheryctl ; then
+    echo "permissions moved to "$WHOAMI
+  else
+  	echo "Cannot move permission. Aborting."
```

Comment:
Good. Can we change to "Unable to write to `/usr/local/bin/mesheryctl`. Please verify write permission for $WHOAMI."?